diff --git a/node_modules/tesseract.js/src/worker-script/index.js b/node_modules/tesseract.js/src/worker-script/index.js
index 4011960..9dfdea0 100644
--- a/node_modules/tesseract.js/src/worker-script/index.js
+++ b/node_modules/tesseract.js/src/worker-script/index.js
@@ -234,7 +234,7 @@ const initialize = async ({
 }, res) => {
   const langs = (typeof _langs === 'string')
     ? _langs
-    : _langs.map((l) => ((typeof l === 'string') ? l : l.data)).join('+');
+    : _langs.map((l) => ((typeof l === 'string') ? l : l.code)).join('+');
 
   const statusText = 'initializing api';
 
